scripts/data/sample.json

```diff
@@ -0,0 +1,4 @@
+[
+    {"name":"Testitem 1","weight":{"value":544},"tags":["Test"],"image":"http://img:3003/serve/test.png"},
```

Comment:
Test data should always represent a real dataset. Let's use either the smartphones or take a few from the sheet.

Reply:
Soll ich nach Daten schauen? Kann ich machen :)

Reply:
Done. Added only the best of data

Reply:
Very nice! Ich würde wenn wir die Daten eh noch aufbereiten range und isCa hinzufügen aber eigenes Issue: https://github.com/world-wide-weights/wwweights/issues/368

Das gehört ja nicht unbedingt zu logic des funktionalität